tests/regression/roles/sonic_prefix_lists/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- debug: msg="sonic_prefix_lists Test started ..."
+
+- name: Preparation test
+  include_tasks: preparation_tests.yaml
+
+- name: "Test {{ module_name }} started ..."
+  include_tasks: tasks_template.yaml
+  loop: "{{ tests }}"
+
+- name: "Cleanup {{ module_name }} started ..."
+  include_tasks: cleanup_tests.yaml
+
+# - name: "Test negative {{ module_name }} started ..."
+#   include_tasks: negative_tasks_template.yaml
+#   loop: "{{ negative_tests }}"
```

Comment:
Should these commented out lines be removed?

Reply:
Yes. I have deleted them.